vscode-extensions/tls-sync-vscode-ui-extension/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode-test/
```

Comment:
Where is this created? Would it be better to put this under a temp directory? We have dedicated `/temp` directories that are ignored that could be use for this (ex. heft plugins have access to dedicated temp dirs for their phase)